kore/app/exec/Main.hs

```diff
@@ -392,6 +445,193 @@ parserInfoModifiers =
                 \in PATTERN_FILE."
     <> header "kore-exec - an interpreter for Kore definitions"
 
+unparseKoreLogOptions :: KoreLogOptions -> [String]
+unparseKoreLogOptions
+    ( KoreLogOptions
+        logType
+        logLevel
+        timestampsSwitch
+        logEntries
+        debugSolverOptions
+        _
+        logSQLiteOptions
+        warningSwitch
+        debugApplyEquationOptions
+        debugAttemptEquationOptions
+        debugEquationOptions
+    )
+  =
+    [ koreLogTypeFlag logType
+    , "--log-level " <> fmap Char.toLower (show logLevel)
+    , timestampsSwitchFlag timestampsSwitch
+    , logEntriesFlag logEntries
+    , debugSolverOptionsFlag debugSolverOptions
+    , logSQLiteOptionsFlag logSQLiteOptions
+    , if warningSwitch == AsError then "--warnings-to-errors" else ""
+    ]
+    <> debugApplyEquationOptionsFlag debugApplyEquationOptions
+    <> debugAttemptEquationOptionsFlag debugAttemptEquationOptions
+    <> debugEquationOptionsFlag debugEquationOptions
```

Comment:
This needs a property test showing that we can parse the output. That test will be easier to write if this function returns a `[String]` equivalent to the list that would be returned by `getArgs`; that is: using separate list entries instead of spaces between arguments.

```suggestion
    concat
    [ koreLogTypeFlag logType
    , ["--log-level", fmap Char.toLower (show logLevel)]
    , timestampsSwitchFlag timestampsSwitch
    , logEntriesFlag logEntries
    , debugSolverOptionsFlag debugSolverOptions
    , logSQLiteOptionsFlag logSQLiteOptions
    , ["--warnings-to-errors" | AsError <- warningSwitch ]
    , debugApplyEquationOptionsFlag debugApplyEquationOptions
    , debugAttemptEquationOptionsFlag debugAttemptEquationOptions
    , debugEquationOptionsFlag debugEquationOptions
    ]
```